### PR TITLE
feat: resending functionality for user invitations and expired password reset links(1.12.dev60)

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### CODE REVIEW

**Review by yuda110 — APPROVED** [review]
LGTM!!

File: apps/web/src/services/auth/password/PasswordPage.vue [hunk 1]
```diff
@@ -12,6 +13,12 @@ import { useI18n } from 'vue-i18n';
 import { useRouter } from 'vue-router';
 import { useStore } from 'vuex';
 
+
+
+import { SpaceRouter } from '@/router';
```
Comment (piggggggggy) [inline apps/web/src/services/auth/password/PasswordPage.vue:18]: nitpick: `SpaceRouter` should be replaced to `const router = useRouter()` in `.vue` components.

File: apps/web/src/services/auth/password/validation-email/ValidationEmailPage.vue [hunk 2]
```diff
@@ -97,12 +97,12 @@ watch(() => route.query.status, (res) => {
                 <div v-if="state.userId"
                      class="collapsible-wrapper"
                 >
-                    <p-collapsible-toggle v-if="!state.isCollapsed"
+                    <p-collapsible-toggle v-if="state.isCollapsed"
                                           v-model="state.isCollapsed"
```
Comment (piggggggggy) [inline apps/web/src/services/auth/password/validation-email/ValidationEmailPage.vue:101]: nitpick: 
```suggestion
                                          v-model="state.is-collapsed"
```

`Kebab` case is rule!

**Review by piggggggggy — APPROVED** [review]
LGTM! 
I left some comments!

**Pull request merged**